packages/datadog-plugin-jest/test/index.spec.js

```diff
@@ -13,6 +13,7 @@ const {
   ERROR_STACK,
   TEST_PARAMETERS
 } = require('../../dd-trace/src/plugins/util/test')
+const { getFormattedJestTestParameters } = require('../src/util')
```

Comment:
Should the tests for this be in its own file? It would make it easier to correlate with the source.

Reply:
done in https://github.com/DataDog/dd-trace-js/pull/1366/commits/abf74e9c598c171c793927fdd5d3bbc7a7068947